### PR TITLE
Translate middleware

### PR DESCRIPTION
**Pull request by jigsawye**
#287
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/laravel-taiwan/docs/pull/301%23discussion_r49834552%22%2C%20%22https%3A//github.com/laravel-taiwan/docs/pull/301%23discussion_r55133132%22%2C%20%22https%3A//github.com/laravel-taiwan/docs/pull/301%23discussion_r55133204%22%2C%20%22https%3A//github.com/laravel-taiwan/docs/pull/301%23discussion_r55300474%22%2C%20%22https%3A//github.com/laravel-taiwan/docs/pull/301%23discussion_r55334441%22%2C%20%22https%3A//github.com/laravel-taiwan/docs/pull/301%23discussion_r55334600%22%2C%20%22https%3A//github.com/laravel-taiwan/docs/pull/301%23discussion_r55335812%22%2C%20%22https%3A//github.com/laravel-taiwan/docs/pull/301%23discussion_r55337758%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%20d8e5c9507de3e2404147d46eee7f9ebcd9ac6750%20middleware.md%20211%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/laravel-taiwan/docs/pull/301%23discussion_r49834552%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Terminable%20%5Cu597d%5Cu96e3%5Cu7ffb...%20%40jigsawye%20%40appleboy%20%5Cu6709%5Cu597d%5Cu60f3%5Cu6cd5%5Cu55ce%5Cr%5Cn%5Cr%5Cnhttp%3A//www.thefreedictionary.com/terminable%5Cr%5Cn%5Cr%5Cn%5Cu770b%5Cu8d77%5Cu4f86%5Cu662f%202.%20Terminating%20after%20a%20designated%20date%22%2C%20%22created_at%22%3A%20%222016-01-15T09%3A27%3A03Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3382565%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/chentsulin%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20middleware.md%3AL204-220%22%7D%2C%20%22Pull%201214a9c6fd020b56a09c9c525a415eba7582d4ea%20middleware.md%2065%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/laravel-taiwan/docs/pull/301%23discussion_r55133132%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%5Cu5be6%5Cu969b%5Cu78b0%5Cu89f8%20%3D%3E%20%5Cu5be6%5Cu969b%5Cu767c%5Cu9001%20%5Cu6703%5Cu6bd4%5Cu8f03%5Cu5bb9%5Cu6613%5Cu7406%5Cu89e3%5Cu55ce%5Cuff1f%22%2C%20%22created_at%22%3A%20%222016-03-06T04%3A40%3A31Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/10325111%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/neighborhood999%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20middleware.md%3AL53-66%22%7D%2C%20%22Pull%201214a9c6fd020b56a09c9c525a415eba7582d4ea%20middleware.md%2071%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/laravel-taiwan/docs/pull/301%23discussion_r55133204%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%5Cu7121%5Cu8ad6%5Cu4e2d%5Cu4ecb%5Cu5c64%5Cu5728%5Cu8acb%5Cu6c42%5Cu524d%5Cu57f7%5Cu884c%5Cuff0c%5Cu6216%5Cu662f%5Cu8acb%5Cu6c42%5Cu5f8c%5Cu57f7%5Cu884c%5Cuff0c%5Cu90fd%5Cu662f%5Cu4f9d%5Cu8cf4%5Cu4e2d%5Cu4ecb%5Cu5c64%5Cu672c%5Cu8eab%5Cu3002%5Cu4f8b%5Cu5982%5Cuff0c...%22%2C%20%22created_at%22%3A%20%222016-03-06T04%3A47%3A01Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/10325111%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/neighborhood999%22%7D%7D%2C%20%7B%22body%22%3A%20%22%5Cu7121%5Cu8ad6%5Cu4e2d%5Cu4ecb%5Cu5c64%5Cu6703%5Cu5728%5Cu8acb%5Cu6c42%5Cu524d%5Cu57f7%5Cu884c%5Cu9084%5Cu662f%5Cu6703%5Cu5728%5Cu8acb%5Cu6c42%5Cu5f8c%5Cu624d%5Cu57f7%5Cu884c%5Cuff0c%5Cu9019%5Cu90fd%5Cu53d6%5Cu6c7a%5Cu65bc%5Cu4e2d%5Cu4ecb%5Cu5c64%5Cu672c%5Cu8eab%5Cu3002%22%2C%20%22created_at%22%3A%20%222016-03-08T09%3A57%3A42Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3382565%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/chentsulin%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20middleware.md%3AL53-66%22%7D%2C%20%22Pull%201214a9c6fd020b56a09c9c525a415eba7582d4ea%20middleware.md%20158%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/laravel-taiwan/docs/pull/301%23discussion_r55300474%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Out%20of%20the%20box%20%3D%3E%20%5Cu9664%5Cu6b64%5Cu4e4b%5Cu5916%5Cr%5Cn%5Cu5176%5Cu5be6%5Cu4e0d%5Cu52a0%5Cu9019%5Cu53e5%5Cu61c9%5Cu8a72%5Cu4e5f%5Cu6c92%5Cu5deeXD%22%2C%20%22created_at%22%3A%20%222016-03-08T00%3A51%3A15Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/10325111%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/neighborhood999%22%7D%7D%2C%20%7B%22body%22%3A%20%22Out%20of%20the%20box%20%5Cu61c9%5Cu8a72%5Cu5c31%5Cu662f%5Cu7ffb%5Cu4f5c%20%5Cu5167%5Cu5efa%22%2C%20%22created_at%22%3A%20%222016-03-08T09%3A27%3A02Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3382565%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/chentsulin%22%7D%7D%2C%20%7B%22body%22%3A%20%22%5Cu89e3%5Cu91cb%202%3A%5Cr%5Cnhttp%3A//www.cnpedia.com/pages/board/view_top.asp%3Fmessageid%3D187%22%2C%20%22created_at%22%3A%20%222016-03-08T09%3A28%3A14Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3382565%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/chentsulin%22%7D%7D%2C%20%7B%22body%22%3A%20%22%40chentsulin%20%5Cu554a%5Cuff01%5Cu6211%5Cu641e%5Cu932f%5Cu7247%5Cu8a9e%5Cu4e86%5Cuff0csorry%22%2C%20%22created_at%22%3A%20%222016-03-08T09%3A39%3A39Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/10325111%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/neighborhood999%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20middleware.md%3AL119-152%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull d8e5c9507de3e2404147d46eee7f9ebcd9ac6750 middleware.md 211'></a> &nbsp;<b><a href='https://github.com/laravel-taiwan/docs/pull/301#discussion_r49834552'>File: middleware.md:L204-220</a></b>
- <a href='https://github.com/chentsulin'></a> Terminable 好難翻... @jigsawye @appleboy 有好想法嗎
http://www.thefreedictionary.com/terminable
看起來是 2. Terminating after a designated date
- [ ] <a href='#crh-comment-Pull 1214a9c6fd020b56a09c9c525a415eba7582d4ea middleware.md 65'></a> &nbsp;<b><a href='https://github.com/laravel-taiwan/docs/pull/301#discussion_r55133132'>File: middleware.md:L53-66</a></b>
- <a href='https://github.com/neighborhood999'></a> 實際碰觸 => 實際發送 會比較容易理解嗎？
- [ ] <a href='#crh-comment-Pull 1214a9c6fd020b56a09c9c525a415eba7582d4ea middleware.md 71'></a> &nbsp;<b><a href='https://github.com/laravel-taiwan/docs/pull/301#discussion_r55133204'>File: middleware.md:L53-66</a></b>
- <a href='https://github.com/neighborhood999'></a> 無論中介層在請求前執行，或是請求後執行，都是依賴中介層本身。例如，...
- <a href='https://github.com/chentsulin'></a> 無論中介層會在請求前執行還是會在請求後才執行，這都取決於中介層本身。
- [ ] <a href='#crh-comment-Pull 1214a9c6fd020b56a09c9c525a415eba7582d4ea middleware.md 158'></a> &nbsp;<b><a href='https://github.com/laravel-taiwan/docs/pull/301#discussion_r55300474'>File: middleware.md:L119-152</a></b>
- <a href='https://github.com/neighborhood999'></a> Out of the box => 除此之外
其實不加這句應該也沒差XD
- <a href='https://github.com/chentsulin'></a> Out of the box 應該就是翻作 內建
- <a href='https://github.com/chentsulin'></a> 解釋 2:
http://www.cnpedia.com/pages/board/view_top.asp?messageid=187
- <a href='https://github.com/neighborhood999'></a> @chentsulin 啊！我搞錯片語了，sorry

<a href='https://www.codereviewhub.com/laravel-taiwan/docs/pull/301?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/laravel-taiwan/docs/pull/301?approve=1'></a>&nbsp;<a href='https://github.com/laravel-taiwan/docs/pull/301'></a>
<a href='#crh-end'></a>

### CODE REVIEW

File: middleware.md [hunk 1]
```diff
@@ -204,16 +204,16 @@ Additional middleware parameters will be passed to the middleware after the `$ne
...
-Middleware parameters may be specified when defining the route by separating the middleware name and parameters with a `:`. Multiple parameters should be delimited by commas:
+在路由中可使用冒號 `:` 來區隔中介層名稱與指派參數，多筆參數可使用逗號作為分隔：
 
     Route::put('post/{id}', ['middleware' => 'role:editor', function ($id) {
         //
     }]);
 
 <a name="terminable-middleware"></a>
-## Terminable Middleware
+## 終止的中介層
```
Comment (chentsulin) [inline middleware.md]: Terminable 好難翻... @jigsawye @appleboy 有好想法嗎

http://www.thefreedictionary.com/terminable

看起來是 2. Terminating after a designated date

File: middleware.md [hunk 2]
```diff
@@ -53,13 +53,13 @@ This command will place a new `OldMiddleware` class within your `app/Http/Middle
 
     }
 
-As you can see, if the given `age` is less than or equal to `200`, the middleware will return an HTTP redirect to the client; otherwise, the request will be passed further into the application. To pass the request deeper into the application (allowing the middleware to "pass"), simply call the `$next` callback with the `$request`.
+如你所見，若是 `age` 小於 `200`，中介層將會回傳 HTTP 重導給用戶端；否則，請求將會進一步傳遞到應用程式。只需呼叫帶有 `$request` 的 `$next` 方法，即可將請求傳遞到更深層的應用程式(允許「通過」中介層)。
 
-It's best to envision middleware as a series of "layers" HTTP requests must pass through before they hit your application. Each layer can examine the request and even reject it entirely.
+HTTP 請求在實際碰觸到應用程式之前，最好是可以層層通過許多中介層，每一層都可以對請求進行檢查，甚至是完全拒絕請求。
```
Comment (neighborhood999) [inline middleware.md]: 實際碰觸 => 實際發送 會比較容易理解嗎？

File: middleware.md [hunk 3]
```diff
@@ -53,13 +53,13 @@ This command will place a new `OldMiddleware` class within your `app/Http/Middle
...
 
-### *Before* / *After* Middleware
+### *前*與*後*中介層
 
-Whether a middleware runs before or after a request depends on the middleware itself. For example, the following middleware would perform some task **before** the request is handled by the application:
+一個中介層是在請求前還是請求後執行要看中介層自己。例如，這個中介層會在應用程式處理請求**前**執行一些任務：
```
Comment (neighborhood999) [inline middleware.md]: 無論中介層在請求前執行，或是請求後執行，都是依賴中介層本身。例如，...

File: middleware.md [hunk 4]
```diff
@@ -119,33 +119,33 @@ If you would like to assign middleware to specific routes, you should first assi
...
 
 <a name="middleware-groups"></a>
-### Middleware Groups
+### 中介層群組
 
-Sometimes you may want to group several middleware under a single key to make them easier to assign to routes. You may do this using the `$middlewareGroups` property of your HTTP kernel.
+有時你可能想將多個中介層組成單一的鍵，讓它可以簡單的指派給路由。你可以使用 HTTP 核心的 `$middlewareGroups` 屬性做到。
 
-Out of the box, Laravel comes with `web` and `api` middleware groups that contains common middleware you may want to apply to web UI and your API routes:
+Laravel 內建了 `web` 及 `api` 中介層群組，包含了你想套用在 web UI 與 API 路由常用的中介層：
```
Comment (neighborhood999) [inline middleware.md]: Out of the box => 除此之外
其實不加這句應該也沒差XD

Comment (chentsulin) [inline middleware.md]: Out of the box 應該就是翻作 內建

Comment (chentsulin) [inline middleware.md]: 解釋 2:
http://www.cnpedia.com/pages/board/view_top.asp?messageid=187

Comment (neighborhood999) [inline middleware.md]: @chentsulin 啊！我搞錯片語了，sorry

File: middleware.md (hunk 3 above)
Comment (chentsulin) [inline middleware.md]: 中介層會在請求前執行還是會在請求後才執行，這取決於中介層本身。

**Pull request merged**